cms/core/models/snippets.py

```diff
@@ -53,3 +58,61 @@ def save(self, *args: Any, **kwargs: Any) -> None:
 
     def __str__(self) -> str:
         return str(self.name)
+
+
+class GlossaryTerm(TranslatableMixin, PreviewableMixin, RevisionMixin, index.Indexed, models.Model):
+    """A model for glossary terms."""
+
+    title = models.CharField(max_length=255)
+    definition = RichTextField(features=settings.RICH_TEXT_BASIC)
+    # to allow for a user to be set on creation and seen in the IndexView
+    updated_by = models.ForeignKey(
```

Comment:
one of the bits I missed from Amelia's notes was 
> A “term owner” field, to manage update requests.


```suggestion
    owner = models.ForeignKey(
"users.User",
        null=True,
        blank=True,
        on_delete=models.SET_NULL,
        related_name="owned_glossary_terms",
    )
    updated_by = models.ForeignKey(
```

+ add to panels, and display on the index view

Reply:
[Added owner field + updated the snippet views](https://github.com/ONSdigital/dis-wagtail/pull/110/commits/a800f0fc6c8661a7c16de3cbc9f0053d01c026a5)